juju/testing/conn.go

```diff
@@ -713,7 +716,7 @@ func AddCharm(st *state.State, curl *charm.URL, ch charm.Charm, force bool) (*st
 		if f, err = os.Open(ch.Path); err != nil {
 			return nil, fmt.Errorf("cannot read charm bundle: %v", err)
 		}
-		defer f.Close()
+		defer func() { _ = f.Close() }()
```

Comment:
Same here ☝️ is there a good reason for the thunk? You got me curious here :)

Reply:
Linter warns about unhandled errors. We're just letting everyone know that we're explicitly ignoring the error.

Reply:
Ah I see, didn't think about the linter 👍 thanks!